<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserItemWidget</class>
 <widget class="QWidget" name="UserItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>49</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>286</width>
    <height>49</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>286</width>
    <height>49</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#cameraStatus{
font-family:&quot;lviconfont&quot;;
font-size:20px;
color:#666666;
background:transparent;
}

QPushButton#micStatus{
font-family:&quot;lviconfont&quot;;
font-size:20px;
color:#666666;
background:transparent;
}

QPushButton#moreSettingBtn
{
font-family:&quot;lviconfont&quot;;
font-size:16px;
color:#666666;
border:none;
background:transparent;
}

QPushButton#firstBtn{
  background: rgba(79,172,255,0.10);
  border: 1 solid #4FACFF;
  border-radius: 2px;
  font-family: &quot;微软雅黑&quot;;
  font-size: 11px;
  color: #4FACFF;
}
QPushButton#firstBtn:hover {
background-color: #1A92FA;
color:white;
}
QPushButton#firstBtn:pressed {
background-color: #1A92FA;
}
QPushButton#firstBtn:disabled {
background-color: #7EADDD;
color: #7AFFFFFF;
}

QPushButton#secondBtn{
  border: 1 solid #eb0011;
  border-radius: 2px;
  font-family: &quot;微软雅黑&quot;;
  font-size: 11px;
  color: #eb0011;
}
QPushButton#secondBtn:hover {
background-color: #ff0040;
color: white;
}
QPushButton#secondBtn:pressed {
background-color: #ff0040;
}
QPushButton#secondBtn:disabled {
background-color: #7EADDD;
color: #7AFFFFFF;
}

QLabel#labelNickname{
  font-family: &quot;微软雅黑&quot;;
  font-size: 12px;
  color: #202020;
}

QLabel#labelStatus{
  font-family: &quot;微软雅黑&quot;;
  font-size: 11px;
  color: #868686;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="labelAvatar">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#labelAvatar{
border-image:url(:/res/images/defalt_avatar.png);
border-radius:2px;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2{
background: rgba(0,0,0,0);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelNickname">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>5400000</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>胡军霖</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelStatus">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>4000000</width>
          <height>15</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>未连麦</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="maximumSize">
      <size>
       <width>44</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">Qwidget#widget_3{
background: rgba(0,0,0,0);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="cameraStatus">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="micStatus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="firstBtn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>InteractiveClass.UserStatus.Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="secondBtn">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>InteractiveClass.UserStatus.Reject</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="moreSettingBtn">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections/>
</ui>
